app/src/main/java/com/infomaniak/euria/MainActivity.kt

```diff
@@ -203,6 +210,11 @@ class MainActivity : ComponentActivity() {
         setTokenToCookie(token)
 
         var currentWebview: WebView? by remember { mutableStateOf(null) }
+        val insets = WindowInsets.safeDrawing
+        val density = LocalDensity.current
+        val layoutDirection = LocalLayoutDirection.current
+
+        applySafeAreaInsetsToWebView(currentWebview, insets, density, layoutDirection)
```

Comment:
Why is this first applySafeAreaInsetsToWebView required? Is the callback not enough?